tests/prepare_test_data.py

```diff
@@ -0,0 +1,132 @@
+import json
+import os
+import shutil
+import tarfile
+import tempfile
+
+import git
+
+from system.plugins.gosymbolextractor.extractor import GoSymbolExtractor
+
+
+DESTDIR = 'testdata'
+
+PROJECTS = [
+    {
```

Comment:
Add comment: if 'export-api' set, generate exported api for project as a part of testdata